src/qmckl_ao.org

```diff
@@ -63,7 +71,16 @@ MunitResult test_qmckl_ao() {
       | ~LMAX(n)~  | input  | Array containing the maximum power for each value |
       | ~P(LDP,n)~ | output | Array containing all the powers of ~X~            |
       | ~LDP~      | input  | Leading dimension of array ~P~                    |
-    
+
+      #+name: args
```

Comment:
I am afraid that the name "args" should be replaced by another name because I think it has to be unique for everything to work.

Reply:
Yes, I agree, ```args``` is not very unique. I suppose we can use the function name itself here by calling ```get_value("Name")``` as the variable name for the table. Does this sound good?